tests/codegen/unwind-extern-imports.rs

```diff
@@ -3,9 +3,22 @@
 
 #![crate_type = "lib"]
 
+// The flag are only at the call sites, not at the declarations.
+// CHECK-LABEL: @force_declare
+#[no_mangle]
+pub unsafe fn force_declare() {
+    // CHECK: call void @extern_fn() [[NOUNWIND_ATTR:#[0-9]+]]
+    extern_fn();
+    // Call without attributes.
+    // FIXME: Make sure that there truly are no attributes!
+    // How can we match "end of line" with FileCheck?
+    // CHECK: call void @c_unwind_extern_fn()
```

Comment:
Here we should check that the line ends after `()`. But how do we do that with FileCheck annotations? I checked the docs but found nothing.

Reply:
Without additional flags, by adding `{{$}}` to the end of the line.

Reply:
Ah, makes sense, thanks!

... and now I am realizing that when I changed this to add attributes in call sites, this call actually did get an attribute.

Reply:
There's no `CHECK-SAME-NOT`, is there? I did this now
```
// CHECK: attributes [[UNWIND_ATTR]]
// CHECK-NOT: nounwind
```
but it feels fragile (if there are ever more attributes this will find a `nounwind` in the next line...)